### PR TITLE
Clone

### PR DESCRIPTION
**Pull request by hector0513**

### CODE REVIEW

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js]:
```suggestion
```

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js:125]:
```suggestion
      case 'ControlLeft':
      case 'ControlRight':
```

we should handle both keyboard keys, just like with ShiftLeft and ShiftRight, just above this ^^

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js:85]:
```suggestion
      case 'ControlLeft':
      case 'ControlRight':
```

we should handle both keyboard keys, just like with ShiftLeft and ShiftRight, just above this ^^

File: src/holochainConfig.js
Comment (Connoropolous) [inline src/holochainConfig.js]:
```suggestion
export const DEFAULT_HOLOCHAIN_HOST = '127.0.0.1'
```

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js]: I think this part should be refactored... 
create a new file somewhere, that exports a function that accepts the necessary arguments and performs these actions... 
e.g. 
`cloneGoals.js`
```javascript

export default function cloneGoals(store, goalsToClone, goalMembers) {
  ...
}
```

**Review by Connoropolous — CHANGES_REQUESTED** [review]
On a great track Hector. 
I have an additional questions for @pegahvaezi 
Copy-Paste is currently cloning 
1. the goals themselves
2. the members/squirrels of the selected goals
Should it also copy the
1. the priority votes?
2. the edges between Goals?

File: src/components/cloneGoals.js
Comment (Connoropolous) [inline src/components/cloneGoals.js]: Does `goals` get used anywhere? remove if not

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js]: Can you move `cloneGoals.js` file into the same folder as this file I guess. `event-listeners`

**Review by Connoropolous — CHANGES_REQUESTED** [review]
Right idea! Just put it in the same folder I guess

File: src/components/cloneGoals.js
Comment (hector0513) [reply src/components/cloneGoals.js]: `.dispatch(
        createGoal.create({
          goal: {
            ...goals[value],
            timestamp_created: moment().unix(),
          },
          maybe_parent_address: null,
        })
      )`

Comment (Connoropolous) [reply src/components/cloneGoals.js]: oh right, I see 👍

File: src/event-listeners/index.js
Comment (pegahvaezi) [reply src/event-listeners/index.js]: re: questions 3 and 4 : no cloning of priority votes (yet) and no copying of edges. If using command + D, the new card should appear the closest possible to the original card.

File: src/reducer.js
Comment (Connoropolous) [inline src/reducer.js]: I know this seems trivial, but can you make the `goal_clone` folder `goal-clone`
Sorry!

**Review by Connoropolous — APPROVED** [review]
I'll change this to approve, but still, if you can make the folder name change before you merge that would be good

**Comment by Connoropolous** [comment]
Pegah says that we don't need to worry about cloning PriorityVotes or Edges yet, so this is fine 👍

File: src/event-listeners/index.js
Comment (Connoropolous) [inline src/event-listeners/index.js:39]:
```suggestion
import { setGoalClone } from '../goal-clone/actions'
```

**Pull request merged**
